puzzle.js

```diff
@@ -85,9 +85,20 @@ puzzleModes["slitherlink"] = {
 }
 
 puzzleModes["solution"] = {
+    "data-drag-draw-edge" : false,
```

Comment:
This shouldn't be necessary, if it is then I've missed a no-input case.

Reply:
Oops, yeah, you're right. It's not needed.